### ISSUE TITLE
Is this possible with Digital Ocean ?

### ISSUE THREAD

**Issue by NotMoni** [serverless-components/express#13]

**Comment by eahefnawy** [comment]
@NotMoni unfortunately, Digital Ocean does not have a serverless offering equivalent to AWS Lambda.

**Comment by NotMoni** [comment]
Awww man. OK
So can i use lambda for my api.website.com and digital ocean for website?

**Comment by NotMoni** [comment]
my API is written with express js

**Comment by eahefnawy** [comment]
Yep, absolutely!

**Comment by NotMoni** [comment]
Awesome.

**Comment by NotMoni** [comment]
@eahefnawy Can i have authenthication on my api so that without the password i wont get charged for a request ?

**Comment by eahefnawy** [comment]
Nope, I don't think that's possible. The authentication logic still has to run somewhere. You could reject the request as soon as possible to save on memory, execution time, and other resources though.

**Comment by NotMoni** [comment]
Oh ok. So with auth how much would it cost per request ?

**Comment by eahefnawy** [comment]
It's really hard to tell. It also depends on the memory setting of your function. But if it's the first thing that executes, I'd imagine you'd be below 100ms, which is the minimum for Lambda pricing:
https://aws.amazon.com/lambda/pricing/
